pkg/aws/core_test.go

```diff
@@ -50,6 +55,11 @@ var _ = Describe("MachineServer", func() {
 			"userData":                []byte("dummy-user-data"),
 		},
 	}
+	annotations := map[string]string{
+		awsPlacementAffinity:         affinity_const,
+		awsPlacementAvailabilityZone: availZone_const,
+		awsPlacementTenancy:          tenancy_const,
```

Comment:
I see that you have test cases with `Default` as a value. Would be nice to have a couple of test cases with customer requirement values of `host`,`host`.